copy/sign.go

```diff
@@ -60,12 +51,24 @@ func (c *copier) createSignature(manifest []byte, keyIdentity string, passphrase
 		}
 	}
 
-	c.Printf("Signing manifest using simple signing\n")
-	newSig, err := signature.SignDockerManifestWithOptions(manifest, identity.String(), mech, keyIdentity, &signature.SignOptions{Passphrase: passphrase})
+	opts := []simple.Option{
+		simple.WithKeyFingerprint(keyIdentity),
+	}
+	if passphrase != "" {
+		opts = append(opts, simple.WithPassphrase(passphrase))
+	}
+	signer, err := simple.NewSigner(opts...)
+	if err != nil {
+		return nil, err
+	}
+	defer signer.Close()
+
+	c.Printf("%s\n", internalSigner.ProgressMessage(signer))
```

Comment:
Note to self: why does this work?  Shouldn't it be `signer.ProgressMessage()`?

Reply:
Ah, OK got it now :heavy_check_mark:

Reply:
What would be a good place to document how this is intended to work?

Reply:
I have no idea.  I think it's OK as is and it was immediately obvious in the code. Just when reading the commits one-by-one I was surprised for a moment.